### DISCUSSION TITLE
Welcome to kiss-vm-ns Discussions!

### DISCUSSION THREAD

**Discussion by tcler** [tcler/kiss-vm-ns#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by lwtcdi** [comment]
Hey My name is Rich.  I am struggling to get the ontap to kvm working.  Does anyone have more info on exactly how this all fits together.

**Reply by tcler** [reply]
Hi Rich
  I don't know if you have seen these tow links? :
  1. https://tcler.github.io/2020/07/27/netapp-ontap-simulator-on-libvirt-kvm/
  2. https://github.com/tcler/ontap-simulator-in-kvm

**Reply by lwtcdi** [reply]
I have seen both links I have the simulator and license files downloaded.  So I am doing this on a MacBook Pro M1.  For the host, (planning on Fedora) do I need to set that up in UTM before I start the process of installing kiss-vm-ns?  

When I try to install kiss-vm-ns part of the git command will work but it errors out when it gets to the  -C (error: unknown switch `C') and another error when the  command gets to "sudo vm prepare" I get this (sudo: vm: command not found).  Any help is appreciated.

**Reply by tcler** [reply]
seems Netapp ONTAP only support x86_64 platform;

**Reply by tcler** [reply]
and kiss-vm-ns only works on Linux(Fedora is recommended), yes you could try install Fedora in UTM and install kiss-vm-ns in Fedora.

**Reply by tcler** [reply]
But as mentioned earlier, ONTAP simulator can not start on MacBook M1/M2 platform; You need an x86_64/Linux platform with >16G ram to get ontap working on KVM.
